proto/kwaque/common/v1/build_info.proto

```
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package kwaque.common.v1;
```

Buf (1.72.0)

[error] Files with package "kwaque.common.v1" must be within a directory "kwaque/common/v1" relative to root but were in directory "proto/kwaque/common/v1".

(PACKAGE_DIRECTORY_MATCH)